### PR TITLE
Corporation tax process

### PR DESCRIPTION
**Pull request by rub1e**
A very long document which takes you through every single step of submitting your own company tax return using the HMRC online service instead of an accountant.

Ref: https://github.com/dwyl/focus-hub/issues/140

### CODE REVIEW

File: corporation-tax.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+* Identify the period you're filing for
+* Filter whatever accounting platform (Xero, FreeAgent) we're using to only include the period you're filing for
+* Ensure all transactions during that period (money in or out) have been reconciled, categorised to the correct account (e.g. office supplies, travel), and with the right VAT rate
+  * In Xero this is easily done by looking at `account transactions` - the column on the right tells you whether something is reconciled or not
+  * In FreeAgent it's slightly trickier - you need to make sure to check each transaction category in the dropdown (screenshot) as FreeAgent has a habit of "hiding" things
+![screenshot 2018-10-16 at 15 44 12](https://user-images.githubusercontent.com/11595920/47025046-bc3a4f80-d15a-11e8-9155-41a17f58e272.png)
+* Make sure you have all the info you need in advance - HMRC logged me out three times while I was looking for something - very frustrating
+  * P&L / Balance sheet reports
+  * Company Reg no (on Companies House page)
+  * Authentication code (in One-Pass)
```
Comment (iteles) [inline corporation-tax.md]: This should absolutely not be in one-pass for our main company (and I don't think it is) - we can eliminate the brackets here.

File: corporation-tax.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  *
```
Comment (iteles) [inline corporation-tax.md]: Stray empty bullet point?

File: corporation-tax.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+* PLEASE BE AWARE: the HMRC system is fiddly and annoying. You are likely to make mistakes like putting the right figure in the wrong box. This is fine - you can go back and edit them as long as you've haven't submitted the final accounts (to be clear, even if you've "approved" the accounts, you can still edit them and approve them again). Just don't be surprised if you try to go back to page 15 but it instead takes you back to page 5 and makes you click through 10 pages to get to where you want to be - this happens a lot.
+  * Basically, edit as much as you like, but only `submit` when you're 100% sure
+
+## HMRC
+
+(This process is written with FocusHub in mind)
+
+* Log in to the relevant HMRC account. The details are in the usual spreadsheet.
+  * NB: 2FA is enabled; the dwyl accounts have @iteles mobile registered, and the FocusHub accounts have @rub1e mobile registered. So best to check the relevant person is available to send you the security code
```
Comment (iteles) [inline corporation-tax.md]: Can we please just make this 'check with @iteles for account holders' as this is likely to change and I don't want to worry about maintenance?

File: corporation-tax.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+  * 
+
+* PLEASE NOTE: **"Figures should be rounded to whole numbers. Round down income and round up spending."** (This pretty essential information is displayed once at the start of the process, and never again.)
+
+* PLEASE BE AWARE: the HMRC system is fiddly and annoying. You are likely to make mistakes like putting the right figure in the wrong box. This is fine - you can go back and edit them as long as you've haven't submitted the final accounts (to be clear, even if you've "approved" the accounts, you can still edit them and approve them again). Just don't be surprised if you try to go back to page 15 but it instead takes you back to page 5 and makes you click through 10 pages to get to where you want to be - this happens a lot.
+  * Basically, edit as much as you like, but only `submit` when you're 100% sure
+
+## HMRC
+
+(This process is written with FocusHub in mind)
```
Comment (iteles) [inline corporation-tax.md]: Given your step by step - let's make this much more strongly worded. WTTE of
> This process is written with FocusHub accounts in mind and is intended _only as an example of what you are likely to encounter_. **Please read _every_ question and answer them correctly for _your_ organisation.**

File: corporation-tax.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+----
+
+* Now just keep hitting `continue` and ignore request for footnotes etc. until you get to the "Approve Accounts" section, then approve the accounts and you can start on the `Computations` section
+
+----
+
+### Computations
+
+NB: there are *loads* of fields which I'm not going to cover because they are obviously not relevant (e.g.  Donations, Vehicle expenses)
```
Comment (iteles) [inline corporation-tax.md]: Not relevant for ***the FocusHub organisation*** and the accounts _we_ filed.

File: corporation-tax.md (hunk 4 above)
Comment (iteles) [reply corporation-tax.md]: We need to make this really stand out as a disclaimer.

File: corporation-tax.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
+# Corporation Tax
```
Comment (iteles) [inline corporation-tax.md]: We need a bold note here around this process being UK only, correct only at the time of writing and maybe a disclaimer (wording suggested below) right up at the top here, to be repeated again below.

File: corporation-tax.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+
+Generally we pay accountants to file our annual company accounts and calculate the Corporation Tax for us.
```
Comment (iteles) [inline corporation-tax.md]: Might be worth adding a sentence here around why this is a sensible practice? Not a blocker.

File: corporation-tax.md (hunk 1 above)
Comment (rub1e) [reply corporation-tax.md]: Removed

File: corporation-tax.md (hunk 2 above)
Comment (rub1e) [reply corporation-tax.md]: Removed

File: corporation-tax.md (hunk 3 above)
Comment (rub1e) [reply corporation-tax.md]: Amended

File: corporation-tax.md (hunk 4 above)
Comment (rub1e) [reply corporation-tax.md]: Added this to the very top - makes sense for it to be the first thing you see

File: corporation-tax.md (hunk 5 above)
Comment (rub1e) [reply corporation-tax.md]: Updated

File: corporation-tax.md (hunk 6 above)
Comment (rub1e) [reply corporation-tax.md]: Done

File: corporation-tax.md (hunk 7 above)
Comment (rub1e) [reply corporation-tax.md]: Done

**Comment by rub1e** [comment]
I've made the request changes but I think I started a `review` by mistake and now I can't close it 😟

**Pull request merged**